_posts/2019-05-21-accessing-kafka-part-5.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title:  "Accessing Kafka: Part 4 - Load Balancers"
+date: 2019-05-13
+author: jakub_scholz
+---
+
+In the fifth and last part of this blog post series we will look at exposing Kafka using Kubernetes Ingress.
+This post will explain how to use Ingress controllers on Kubernetes, how does Ingress compare with OpenShift Routes and how it can be used with Strimzi and Kafka.
```

Comment:
```suggestion
This post will explain how to use Ingress controllers on Kubernetes, how Ingress compares with OpenShift Routes and how it can be used with Strimzi and Kafka.
```